ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1760,6 +1760,56 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-ingress-glb
+- as: gcp-upi-private-xpn-sdn-amd-f28-destructive
+  cron: 14 2 20 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      E2E_RUN_TAGS: '@gcp-upi'
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn
+- as: gcp-upi-private-xpn-sdn-arm-f14
+  cron: 29 11 10,25 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BOOTSTRAP_NODE_TYPE: t2a-standard-4
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      CONTROL_PLANE_NODE_TYPE: t2a-standard-4
+      E2E_RUN_TAGS: '@gcp-upi'
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
```

Comment:
OK

Reply:
BTW, no need to prefix `ocp-e2e` and suffix `cucushift` for the test name.
The test name is limited to 63 chars, so it's better to use a short and simple name style the team is currently using.